qpsolvers/solvers/__init__.py

```diff
@@ -359,6 +359,50 @@
 except ImportError:
     pass
 
+# KVXOPT
+# ======
+
+kvxopt_solve_problem: Optional[
+    Callable[
+        [
+            Problem,
+            Optional[str],
+            Optional[ndarray],
+            bool,
+        ],
+        Solution,
+    ]
+] = None
+
+kvxopt_solve_qp: Optional[
+    Callable[
+        [
+            Union[ndarray, csc_matrix],
+            ndarray,
+            Optional[Union[ndarray, csc_matrix]],
+            Optional[ndarray],
+            Optional[Union[ndarray, csc_matrix]],
+            Optional[ndarray],
+            Optional[ndarray],
+            Optional[ndarray],
+            Optional[str],
+            Optional[ndarray],
+            bool,
+        ],
+        Optional[ndarray],
+    ]
+] = None
+
+try:
+    from .kvxopt_ import kvxopt_solve_problem, kvxopt_solve_qp
```

Comment:
The CI coverage job doesn't pass because of a ruff error:

```
$ ruff check qpsolvers
qpsolvers/solvers/__init__.py:397:48: F401 `.kvxopt_.kvxopt_solve_qp` imported but unused; consider using `importlib.util.find_spec` to test for availability
    |
396 | try:
397 |     from .kvxopt_ import kvxopt_solve_problem, kvxopt_solve_qp
    |                                                ^^^^^^^^^^^^^^^ F401
398 |
399 |     solve_function["kvxopt"] = kvxopt_solve_problem
    |
    = help: Remove unused import: `.kvxopt_.kvxopt_solve_qp`
```

This one is just because we need to add "kvxopt_solve_qp" to the `__all__` list at the end of the file.